VeroliBGV/BGV008.xml

```diff
@@ -0,0 +1,836 @@
+                           </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.5.4">
+                           <locus from="164ra" to="166rb"/>
+                           <title type="complete" ref="LIT2509Weddas#Thursday"/>
+                           <incipit xml:lang="gez">
+                              <locus target="#164ra"/>
+                              <hi rend="rubric">ውዳሴ፡ ዘሐሙስ፡ ዕፀ፡ እንተ፡</hi> ርእየ፡ ሙሴ፡ በነደ፡ እሳት፡ ውስተ፡ ገዳም፡ ወአዕፁቂሃ፡ ኢትውኢ፡ ትመስል፡ <hi rend="rubric">ማርያም፡</hi> 
+                              ድንግል፡ ዘእንበለ፡ ርኩስ።
+                           </incipit>
+                           <explicit xml:lang="gez">
+                              <locus target="#166rb"/>
+                              ወተወልደ፡ በቤተ፡ ልሔም፡ በከመ፡ ሰበኩ፡ ነቢያት፡ አድኃነነ፡ ወቤዘወነ፡ ወረሰየነ፡ ሕዝበ፡ ዚአሁ፡ ሰ፡ ቅ።
+                           </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.5.5">
+                           <locus from="166rb" to="167vb"/>
+                           <title type="complete" ref="LIT2509Weddas#Friday"/>
+                           <incipit xml:lang="gez">
+                              <locus target="#166rb"/>
+                              <hi rend="rubric"><surplus>ው</surplus>ው<add place="interlinear">ዳ</add>ሴ፡ </hi> ዘዓርብ፡ ቡርክት፡ አንቲ፡ እምአንስት፡ ወቡሩክ፡ ፍሬ፡ 
+                              ከር<supplied reason="omitted" resp="MV">ሥ</supplied>ኪ፡ ኦ<hi rend="rubric">ማርያም፡</hi> 
+                              ድንግል፡ ወላዲተ፡ አምላክ፡ ዘእንበለ፡ ርኵስ፡ ሠረቀ፡ ለነ፡ እምኔኪ፡ ፀሐየ፡ ጽድቅ።
+                            </incipit>
+                           <explicit xml:lang="gez">
+                              <locus target="#167vb"/>
+                              በ<supplied reason="omitted" resp="MV">እ</supplied>ንተዝ፡ ንዌድሰኪ፡ በከመ፡ ገብርኤል፡ መልአክ፡ እንዘ፡ ንብል፡ ቡርክት፡ አንቲ፡  <hi rend="rubric">ማርያም፡</hi> 
+                              ወቡሩክ፡ ፍሬ፡ ከርሥኪ፡ ተፈሥሒ፡ ኦምልዕተ፡ ጸጋ፡ እ<hi rend="ligature">ግዚ</hi>አብሔር፡ ምስሌኪ። ሰ፡ ቅ።
+                           </explicit>
+                         </msItem>
+                        <msItem xml:id="ms_i1.5.6">
+                           <locus from="167vb" to="168vb"/>
+                           <title type="complete" ref="LIT2509Weddas#Saturday"/>
+                           <incipit xml:lang="gez">
+                              <locus target="#167vb"/>
+                              <hi rend="rubric">ውዳሴ፡</hi> ዘቀዳሚት፡ ሰንበት፡ ንጽሕት፡ ወብርህት፡ ወቅድስት፡ በኵሉ፡ እንተ፡ ሐቀፈቶ፡ ለእ<hi rend="ligature">ግዚ</hi>እ፡ በዕራኃ፡ ወኵሉ፡ ፍጥረት፡ 
+                              ይትፌሥሑ፡ ምስሌሃ፡ እንዘ፡ ይጸርሑ፡ ወይብሉ፡ ሰ።
+                           </incipit>
+                           <explicit xml:lang="gez">
+                              <locus target="#168vb"/>
+                              ዘኢ<supplied reason="omitted" resp="MV">ይ</supplied>ትጸንን፡ ምስማኮሙ፡ ለቅዱሳን፡ ሰአሊ፡ ለነ፡ ኃበ፡ ወልድኪ፡ ኄር፡ መድኃኒነ፡ 
+                              ይምሐረነ፡ ወይሠሃለነ፡ ይስረይ፡ ኃጢአትነ፡ ሰ።
```

Comment:
ኃጢኣትነ፡ (Massimo io continuo a segnalarti questi plurali, ma ora comincio a pensare che se li hai scritti così c'è una ragione. Se così fosse, scusami e ignora anche le segnalazioni precedenti)

Reply:
For Sanne: I just realised that I keep on correcting these plurals but maybe Massimo had a good reason to have them like that. If this is the case, he shall ignore my previous messages as well